parts/linux/cloud-init/artifacts/ci-syslog-watcher.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o pipefail
+
+[ ! -f "/var/run/mdsd-ci/update.status" ] && exit 0
+status=$(cat /var/run/mdsd-ci/update.status)
+
+if [[ "$status" == "add" ]]; then
+        echo "Status changed to $status."
+        [ -f "/var/run/mdsd-ci/70-rsyslog-forward-mdsd-ci.conf" ] && cp /var/run/mdsd-ci/70-rsyslog-forward-mdsd-ci.conf /etc/rsyslog.d
+elif [[ "$status" == "remove" ]]; then
+        echo "Status changed to $status."
+        [ -f "/etc/rsyslog.d/70-rsyslog-forward-mdsd-ci.conf" ] && rm /etc/rsyslog.d/70-rsyslog-forward-mdsd-ci.conf
```

Comment:
do we run into issue where we are try to remove or add the file while this file being used by rsyslog?

Reply:
there is no such issue. rsyslog loads the configs on startup and is not tied to the files on disk. the conf files can be safely edited.